### DISCUSSION TITLE
Can't use Nodejs 16 anymore with Gatsby

### DISCUSSION THREAD

**Discussion by agtz92** [gatsbyjs/gatsby#37055] [Help]
I had to modify some settings on a website built earlier this year and unfortunately, after a long research it turns out that my issue had to do with my nodejs version which was 16.x.x . 
After updating to Nodejs 18 everything worked out ok, however, Netlify and many other similar options do not support Nodejs 18 just yet because AWS is still working on that.

My question is, how can I make gatsby work with NodeJS 16 again? It seems like I will not be able to upload them to almost any server until NodeJS 18 is supported by AWS

Thanks

**Comment by LekoArts** [comment]
Gatsby 5 only supports Node.js 18
